buildSrc/src/main/kotlin/vauhtijuoksu-api.kotlin-common-conventions.gradle.kts

```diff
@@ -28,7 +30,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
 
     // Use JUnit Jupiter API for testing.
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.2")
+    testImplementation("org.junit.jupiter:junit-jupiter-api")
```

Comment:
Why the version is removed here but not at line 23?

Reply:
It's in the constraints block, which sets the version for all the submodules we might end up using.